plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestSplitPruning.java

```diff
@@ -358,6 +359,33 @@ public void testParquetStatisticsPruning()
         testCountQuery("SELECT count(*) FROM parquet_struct_statistics WHERE row = ROW(2, 'b')", 3, 9);
     }
 
+    @Test
+    // See README.md for table values
```

Comment:
I don't think code readers want to look for README file when seeing this test.

Reply:
I have copied it as https://github.com/trinodb/trino/pull/17160/files/7695b9a72e0122cc99bffe66ed47a03482a205da#diff-ec00db9ee4bb8d02ce63d04b35bf788c311afef51b7c56bdf23ab5ad8dc5101eR341

Okat

Reply:
@krvikash I meant I didn't want code readers waste their time to look for the README file. When we see **_only_** `testPrimitiveFieldsInsideRowColumnPruning` test, we can't understand why these values are expected. 

Anyway, I'm going to merge this PR as-is.